pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -145,13 +163,34 @@ func (vs *VSphere) HasClusterID() bool {
 // Initializes vSphere from vSphere CloudProvider Configuration
 func buildVSphereFromConfig(cfg *CPIConfig) (*VSphere, error) {
 	nm := newNodeManager(cfg, nil)
-
+	lb, err := loadbalancer.NewLBProvider(&cfg.LBConfig)
+	if err != nil {
+		return nil, err
+	}
+	if _, ok := os.LookupEnv("ENABLE_ALPHA_NSXT_LB"); !ok {
+		if lb != nil {
+			klog.Infof("To enable NSX-T load balancer support you need to set the env variable ENABLE_ALPHA_NSXT_LB")
+			lb = nil
+		}
+	} else {
+		if lb == nil {
+			return nil, fmt.Errorf("To enable NSX-T load balancer support you need to configure section LoadBalancer")
+		}
+	}
+	if lb == nil {
+		klog.Infof("NSX-T load balancer support disabled")
+	} else {
+		klog.Infof("NSX-T load balancer support enabled")
```

Comment:
I second this as well

Reply:
Changed logging to `klog.Infof("NSX-T load balancer support enabled. This feature is alpha, use in production at your own risk.")`. Also added the sentence "since this is an alpha feature... can change." to the README.md.